apps/builder/app/builder/features/topbar/publish.tsx

```diff
@@ -554,24 +571,40 @@ const ExportContent = () => {
             Step 3
           </Text>
           <Text color="subtle">
-            Use the{" "}
+            Run this command to publish to{" "}
             <StyledLink
-              href="https://vercel.com/docs/cli"
+              href={DeployTargetDocs[deployTarget]}
               target="_blank"
               rel="noreferrer"
             >
-              Vercel CLI
+              {humanizeString(deployTarget)}
             </StyledLink>{" "}
-            to publish your project on Vercel.
           </Text>
         </Grid>
+
+        <Select
+          fullWidth
+          css={{ zIndex: Number.MAX_SAFE_INTEGER.toString() }}
```

Comment:
why?

Reply:
This select is on top of a existing popup. So, it's going behind the popup.

Reply:
seems like select should be using portal inside, we use portal 99% of the time to solve zindex issues

Reply:
ah no its already using a portal, then the question is what is publish popover doing to be higher zIndex

Reply:
seems like popover is using zIndex: theme.zIndices[1], in publish.tsx, so probably would be just using theme.zIndices[2]